hdf5-sys/src/h5o.rs

```diff
@@ -44,6 +44,17 @@ pub const H5O_HDR_ALL_FLAGS: c_uint = H5O_HDR_CHUNK0_SIZE
 pub const H5O_SHMESG_MAX_NINDEXES: c_uint = 8;
 pub const H5O_SHMESG_MAX_LIST_SIZE: c_uint = 5000;
 
+pub const H5O_INFO_BASIC: c_uint = 0x0001;
```

Comment:
Have these constants remained the same throughout HDF5 versions? Just wanted to confirm that it would be the same from 1.8.5 to 1.10.5.

Reply:
I went through the code, verified these were unchanged an introduced in version 1.10.3. Added cfg attribute.

Reply:
Thanks! I must have missed it when updating.